kf_update_dbgap_consent/app/cli.py

```diff
@@ -46,12 +46,25 @@ def cli():
             " - Defaults to match on `external_sample_id`"
         ),
     )
+    parser.add_argument(
+        "--coerce_visible",
+        action="store_true",
+        default=False,
+        help=(
+            "If a specimen is loaded into dbgap, set the specimens, its "
+            "descendants, and the associated participant to visible."
```

Comment:
Descendants of the specimen, not of the participant? What about phenotypes, diagnoses, and family relationships?

Reply:
That's a good point. 

I need to pull in `non-otherspecimen descendants of the participant`